javascript/javascript1/week4/js1-week4-homework/index.js

```diff
@@ -0,0 +1,92 @@
+const name = [];
+const toDo = [];
+function getReply(command) {
+  const lowerCase = command.toLowerCase();
+  //console.log(lowerCase)
+
+  let words = lowerCase.split(" ");
+  let myName = words[words.length - 1];
+  name.push(myName);
+  //console.log(name)
+  if (lowerCase.includes("hello my name is")) {
+    console.log("Nice to meet you " + name[0]);
+  }
+  if (lowerCase.includes("what is my name")) {
+    console.log("Your name is " + name[0]);
+    /* function whatIsYourName(){
+        console.log("Your name is "+name)
+       }
+         whatIsYourName(); */
+  }
+  if (lowerCase.includes("add fishing to my todo")) {
+    toDo.push("fishing");
+    console.log("fishing added to your todo");
+  }
+  if (lowerCase.includes("add singing in the shower to my todo")) {
+    toDo.push("singing in the shower");
+    console.log("singing in the shower added to your todo");
+  }
+  if (lowerCase.includes("remove fishing from my todo")) {
+    toDo.shift("fishing");
+    console.log("Removed fishing from your todo");
+  }
```

Comment:
Bonus: These commands could be general, I should be able to write "add code review to my todo" and it should work the same way. 

So you'll need to catch the pattern "add TASK to my todo", extract TASK which could be multiple words, and push that into the todo. 
Similar for remove, you need to find "remove TASK from my todo", then find and remove that task from the list if it's in there.